tests/integration/test_app_create.py

```diff
@@ -124,6 +124,7 @@ def test_csp_policy_headers(self):
             "EQ_ENABLE_LIVE_RELOAD": False,
             "CDN_URL": cdn_url,
             "ADDRESS_LOOKUP_API_URL": address_lookup_api_url,
+            "EQ_APPLICATION_VERSION": "v1.0.0",
```

Comment:
Unrelated change?, Will need a `application-version` file as per README.

Reply:
Removed, and added `.application-version`, which kept the coverage to 100%.